airbyte-integrations/connectors/source-amplitude/acceptance-test-config.yml

```diff
@@ -1,24 +1,42 @@
+# See [Connector Acceptance Tests](https://docs.airbyte.io/connector-development/testing-connectors/connector-acceptance-tests-reference)
+# for more information about how to configure these tests
 connector_image: airbyte/source-amplitude:dev
-tests:
+test_strictness_level: high
+acceptance_tests:
   spec:
-    - spec_path: "source_amplitude/spec.json"
-      backward_compatibility_tests_config:
-        disable_for_version: "0.1.17"
+    tests:
+      - spec_path: "source_amplitude/spec.yaml"
   connection:
-    - config_path: "secrets/config.json"
-      status: "succeed"
-    - config_path: "integration_tests/invalid_config.json"
-      status: "failed"
+    tests:
+      - config_path: "secrets/config.json"
+        status: "succeed"
+      - config_path: "integration_tests/invalid_config.json"
+        status: "failed"
   discovery:
-    - config_path: "secrets/config.json"
+    tests:
+      - config_path: "secrets/config.json"
+        backward_compatibility_tests_config:
+          disable_for_version: 0.1.21
```

Comment:
can you clarify why we're disabling backwards compatibility tests and how this impacts users in production?

Reply:
stream state and cursor field for stream event has been changed.